docs/environment/README.md

```diff
@@ -0,0 +1,44 @@
+# Environment
+
+This project can be run in two different environments: development and production.
+
+## Requirements
+
+In order to use MathType and ChemType for editing and generate mathematical expressions, you need to install this plugin along with one of the following:
+
+- [MathType for Atto](https://github.com/wiris/mooodle-atto_wiris)
+- [MathType for TinyMCE 3.x](https://github.com/wiris/moodle-tinymce_tiny_mce_wiris) for Moodle versions under 4.2.
+- [MathType for TinyMCE 6.x](https://github.com/wiris/moodle-tinymce6_wiris) for Moodle versions above 4.2.
+
+## Development
+
+### Install MathType filter
+
+It is necessary to have a [Moodle development environment](https://github.com/moodlehq/moodle-docker) configured. 
+
+Install the plugin like any other plugin to folder `/filter/wiris`.
+
+You can use git:
+
+```sh
+$ git clone https://github.com/wiris/moodle-filter_wiris.git filter/wiris
+```
+
+Alternatively, you can [download the plugin](https://github.com/wiris/moodle-filter_wiris/archive/main.zip) and unzip the file into filters folder, then rename the new folder to `wiris`.
+
+### Uninstall MathType filter
+
+This project is a dependency of the following:
+
+* MathType for Atto
+* MathType for TinyMCE
+* MathType for WirisQuizzes
+
+
+To uninstall the Wiris filter in Moodle, all the previous plugins mentioned must also be uninstalled. If any of these plugins are still present, the MathType filter cannot be removed.
+
+## Production
+
+Follow this [guideline offered by Moodle](https://docs.moodle.org/405/en/Installing_plugins#Installing_a_plugin) to learn how to install any plugin to a production environment.
+
+The [MathType filter](https://moodle.org/plugins/filter_wiris) wiris can be found on the Moodle plugins directory, or click on the previous link.
```

Comment:
Las instrucciones en este documento son para el entorno de desarrollo, estas que tu pones yo considero que en todo caso iría más a USAGE.

Reply:
Según el documento de herodotos, las instrucciones para instalar el paquete en producción y staging también van en este documento

Reply:
No, en este documento se incluyen las instrucciones para, literalmente:

"**Explain step by step how to build the Production Environment**. If there is not a Production Environment, please alert about it.

The contents of this section can be placed in an external document following Ogmios guidelines. In that case, please add here a link to that document."

En ningún momento se habla de instalar el plugin en el entorno de desarrollo, eso como te decía pertenece em todo caso a USAGE, donde pone:

 "Usage is focused on providing details about how to use this project from a user point of view. This document is not a replacement or duplication of docs.wiris.com."
 
 y que creo que se ajusta más a la información que tu estás incluyendo aqui 👀

Reply:
¡Entendido! Arreglado 👍🏻